editor/src/components/canvas/canvas-utils.spec.browser2.tsx

```diff
@@ -207,8 +207,9 @@ describe('updateFramesOfScenesAndComponents - pinFrameChange -', () => {
 
   it('no layout prop on child', async () => {
     const renderResult = await renderTestEditorWithCode(
+      // FIXME This should be fine with {{ ...props.style }} but that causes the uids to switch in tests only
```

Comment:
This one is beyond me. The issue only shows up in this specific test, and only shows up when running in the test environment